pages/careers/_open-positions.html

```diff
@@ -11,24 +11,28 @@ <h2 class="bold">Open Positions</h2>
 
     <ul>
       <li><a href="#senior-software-engineer-developer-experience-team">Senior Software Engineer - Developer Experience Team</a></li>
+      <li><a href="#senior-software-engineer-brand-new-product">Senior Software Engineer - Brand New Product</a></li>
       <li><a href="#senior-software-engineer-contract-part-time">Senior Software Engineer (Contract/Part-Time)</a></li>
       <li><a href="#engineering-manager">Engineering Manager</a></li>
+      <li><a href="#product-manager-iac">Senior Product Manager - Infrastructure as Code</a></li>
+      <li><a href="#product-manager-brand-new-product">Senior Product Manager - Brand New Product</a></li>
+      <li><a href="#designer-brand-new-product">Senior Designer - Brand New Product</a></li>
     </ul>
```

Comment:
And here it is with the "Senior" designation removed:

<img width="588" alt="image" src="https://user-images.githubusercontent.com/4295964/165018675-1392f83d-e5f7-4db6-a69d-58b949d8a654.png">

Reply:
> I think we need to be consistent with our meta data here. Right now, the first role says "[position] - ." The second role says "[position] - New Product." The third role says "[position] (part-time)." The fourth says "[position]." The fifth says "[position] - product area."

Ah, good point. Standardizing is a good idea 👍 

> I also noticed that a few places don't bother with titles like senior or principal or junior, and instead mark some roles as "Early Career."

I'm not a fan of this. If the positions are senior, then we don't want a bunch of junior people applying, wasting their time and ours. 

> I think our products actually work better for the point of these job listings. So that would look something like this:
> 
> <img alt="image" width="630" src="https://user-images.githubusercontent.com/4295964/165017538-c0304f74-9ae9-4f21-a5ee-6dd35abd2705.png">
> 
> I think that's my favorite formulation so far. What do you think?

+1 on products. I noticed that it says "New Team" for one of them, which should be "New Product," but otherwise, that LGTM. I think the PMs, EMs, and Designer should have products listed too. I see no reason not to?

Update with standardized titles is here: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/710/commits/50e73dda17beab5b077acc47646e2f94b928d77b